arviz/plots/backends/bokeh/densityplot.py

```diff
@@ -66,18 +66,17 @@ def plot_density(
     if data_labels is None:
         data_labels = {}
 
+    legend_items = {}
     for m_idx, plotters in enumerate(to_plot):
-        for ax_idx, (var_name, selection, values) in enumerate(plotters):
+        for _, (var_name, selection, values) in enumerate(plotters):
```

Comment:
the enumerate an be removed completely if the index is not needed.

Reply:
Yes, makes sense.